apps/andi/lib/andi/migration/date_coercer.ex

```diff
@@ -0,0 +1,40 @@
+defmodule Andi.Migration.DateCoercer do
+  @moduledoc """
+  Either parse to valid iso8601 or convert to empty string.
+  """
+  def coerce_date(date) do
+    case DateTime.from_iso8601(date) do
+      {:ok, _parsed_date, _offset} -> date
+      _ -> fix_date(date)
+    end
+  end
+
+  def fix_date(""), do: ""
+
+  def fix_date(date) do
+    format_strings = [
```

Comment:
Did we decide to not check for d/m/y dates?

Reply:
Added.